UI/forms/OBSBasicSettings.ui

```diff
@@ -7855,7 +7851,6 @@
   <tabstop>simpleOutRecFormat</tabstop>
   <tabstop>simpleOutRecEncoder</tabstop>
   <tabstop>simpleOutMuxCustom</tabstop>
-  <tabstop>simpleReplayBuf</tabstop>
```

Comment:
Missing tabstop, so users might not be able to keyboard-navigate to the settings?

Reply:
You're right, initially this was removed when the group box kept its name (when I then only changed later).
Undone.